todo_app/data/trello_items.py

```diff
@@ -0,0 +1,184 @@
+from flask import session
+import os
+import json
+import requests
+
+# There is currently an issue with https certificate verification, switch to True once fixed
+_HTTPS_VERIFY_CERTIFICATES = False
+
+class Item:
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list['name'])
+
+url = "https://api.trello.com/1/boards/" + str(os.environ.get('BOARD_ID'))
+
+headers = {
+  "Accept": "application/json"
+}
+
+_DEFAULT_ITEMS = [
```

Comment:
It looks like this is unused leftover from session_items - lets get rid of it, we can always recover it with Git later! On that note, clearing out the entire `session_items.py` file at this point feels like sensible maintenance